Tests/Controller/CookieConsentControllerTest.php

```diff
@@ -52,7 +52,7 @@ public function setUp(): void
         $this->templating              = $this->createMock(\Twig_Environment::class);
         $this->formFactory             = $this->createMock(FormFactoryInterface::class);
         $this->cookieChecker           = $this->createMock(CookieChecker::class);
-        $this->translator              = $this->createMock(TranslatorInterface::class);
+        $this->translator              = $this->createMock(Translator::class);
```

Comment:
Should be https://github.com/symfony/translation-contracts/blob/master/TranslatorInterface.php IMHO.

Reply:
That interface doesnt have a setlocale method